src/storage.cc

```diff
@@ -314,6 +318,62 @@ Status Storage::RestoreFromBackup() {
   return s.ok() ? Status::OK() : Status(Status::DBBackupErr, s.ToString());
 }
 
+Status Storage::Reopen() {
+  auto s = Open();
+  if (!s.IsOK()) {
+    LOG(ERROR) << "[storage] Fail to reopen db, error: " << s.Msg();
+    LOG(ERROR) << "[storage] Exiting...";
+    exit(1);
```

Comment:
the old backup method will restart fullsync in this situation,  restart seems  better?

Reply:
yes, I will fix